### PR TITLE
Add travis support

### PR DESCRIPTION
**Pull request by liztio**
Signed-off-by: liz <liz@heptio.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,22 +50,24 @@ KUBECFG_CMD = $(DOCKER) run \
 	$(KSONNET_BUILD_IMAGE) \
 	kubecfg show -o yaml -V RBAC_ENABLED=$(RBAC_ENABLED) -J $(WORKDIR) -o yaml $< > $@
 
+DOCKER_RUN ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
```
Comment (timothysc) [inline Makefile]: Might want to change the alias name.  DOCKER_BUILDER

File: Makefile [hunk 2]
```diff
@@ -50,22 +50,24 @@ KUBECFG_CMD = $(DOCKER) run \
...
+
 .PHONY: all container push clean cbuild test local generate
 
 all: container
 
 test:
-	$(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c '$(TEST)'
+	 $(DOCKER_RUN) '$(TEST)'
```
Comment (timothysc) [inline Makefile]: want to have the ```test: cbuild``` 

and you could get rid of the other stage above.

Comment (liztio) [reply Makefile]: I don't quite understand this. They are running different commands?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+language: go
+services:
+  - docker
+install: true
+
+jobs:
+  include:
+    - stage: test
+      script: make test
+    - script: make cbuild
```
Comment (chuckha) [inline .travis.yml]: Is this a separate stage with no name?

File: Makefile (hunk 2 above)
Comment (timothysc) [reply Makefile]: They are, but you probably shouldn't run the tests if the builds fail. 

So making it a dependent target makes sense to me.

File: .travis.yml (hunk 3 above)
Comment (liztio) [reply .travis.yml]: Unnamed stages inherit the previous stage's name, so you can have multiple simultaneous jobs.

File: Makefile (hunk 2 above)
Comment (liztio) [reply Makefile]: legit!

File: Makefile [hunk 4]
```diff
@@ -50,22 +50,24 @@ KUBECFG_CMD = $(DOCKER) run \
...
 
-container: cbuild
+container: test cbuild
```
Comment (timothysc) [inline Makefile]: if you do the ```test: cbuild``` change 

then this can become ```container: test```

**Comment by timothysc** [comment]
@chuckha now to switch the default checks on the repo and setup the webhooks.

Also nix a jenkins job!  It will make @davecheney smile ;-)

**Pull request merged**